index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
I think you should change the title tag. Make sure it is related to what is on your page, in other words, it should give your page visitor a hint on what is on the page since its what appears on the browser.